js/repl/compile.ts

```diff
@@ -167,6 +168,8 @@ export default function compile(code: string, config: CompileConfig): Return {
         : undefined,
     };
 
+    console.info("Babel config", babelConfig);
```

Comment:
Missed this

Reply:
Yeah, that's fine for me. I don't mind removing it though.

Reply:
Sorry, I meant "you forgot to remove this" :sweat_smile: 
Or is it intentional? I often use `console.log` in the repl, and I am worried that this would produce a lot of logs.

Reply:
This is intentional, but I don't mind removing it at all.
Also I thought this should be `console.debug`, which I forgot about at first. :)

Reply:
Ok, I would prefer to hide it. Also because the `@babel/standalone` config format does not match the normal config format (for example, plugin/preset names), so people wouldn't be able to copy-paste it anyway.

I'm +1 for showing the generated/normalzied json config, but that should probably happen in a way more integrated with the UI.

Reply:
Yes, I want to improve it a bit, but it's a little difficult.